components/hab/src/command/studio/README.md

```diff
@@ -0,0 +1,43 @@
+## Native Packages
+
+Native Packages are a new Habitat feature. They enable the user to run and manage applications with
+the Habitat Supervisor on platforms which do not have package support.
+
+Native Packages enable the following capabilities:
+- User can create a package for any unix platform that has support for the Habitat Supervisor. Eg: M1 Macs, ARM, x86_64
+- User can define hooks that can be run by a native interpreter other than bash. Eg: You could have node js or python hooks that get executed by a native node js or python interpreter in your runtime environment.
+
+Native Packages have the following limitations:
+- No depedency on other standard or native packages
+- No runtime environment gaurantee. You must ensure that all required libraries and executables are available in the environment the supervisor runs the package.
+- You currently cannot upload or download native packages from the Habitat Builder. You must copy the built hart file to the production runtime environment through your own deployment mechanisms.
```

Comment:
we deployed the builder support. We should include instructions on how enable the native package support on builder.

Reply:
Addressed in commit da089ad